utils/savemodel.py

```diff
@@ -0,0 +1,8 @@
+import logging
```

Comment:
I don't understand why you created a new file in utils for one line of code. Plus the session.commit is into another place.